packages/gatsby-remark-code-repls/src/index.js

```diff
@@ -114,17 +126,22 @@ module.exports = (
                   }
                   return map
                 }, {}),
+                main: filesPaths[0].fileName,
```

Comment:
I think we should actually use the full path, not just file name. Might be conflicts otherwise.

Reply:
> Non-CodeSandbox REPL targets should also error if given this syntax, since they don't support it.

My changes are all covered in `if (node.url.startsWith(PROTOCOL_CODE_SANDBOX))` so I don't think it'll affect other platforms.

> I think we should actually use the full path, not just file name. 

I am not sure here. Do you mean we put the normalized path in the main field and the name of each files? I don't think it's gonna work that way bc codesandbox won't be able to recognize the full path in remote.

Reply:
> My changes are all covered in `if (node.url.startsWith(PROTOCOL_CODE_SANDBOX))` so I don't think it'll affect other platforms.

Agreed that your changes don't impact other platforms, but we should warn if someone uses the unsupported syntax for other platforms.

> I am not sure here. Do you mean we put the normalized path in the main field and the name of each files?

I mean the relative path people pass (e.g. `path/foo.js`)

Reply:
Sorry I misunderstood what you meant. It makes sense now. Thanks!